prefect_aws/credentials.py

```diff
@@ -21,10 +19,20 @@ class AwsCredentials:
             This is only needed when you are using temporary credentials.
         profile_name: The profile to use when creating your session.
         region_name: The AWS Region where you want to create new connections.
+
+    Example:
+        Load stored AWS credentials:
+        ```python
+        from prefect_aws import AwsCredentials
+
+        aws_credentials_block = AwsCredentials.load("MY_BLOCK_NAME")
     """  # noqa E501
 
+    _block_type_name = "AWS Credentials"
+
+
     aws_access_key_id: Optional[str] = None
-    aws_secret_access_key: Optional[str] = None
+    aws_secret_access_key: Optional[SecretStr] = None
```

Comment:
Below, since this is now a SecretStr, need to `get_secret_value` like:
 `aws_secret_access_key=self.aws_secret_access_key.get_secret_value(),`

Reply:
Oh, nice catch!

Reply:
Haha only because I had to make another PR to fix missing get_secret_value recently!